lang/python/airr/interface.py

```diff
@@ -196,3 +200,148 @@ def validate_rearrangement(filename, debug=False):
     handle.close()
 
     return valid
+
+
+def load_repertoire(filename, validate=False, debug=False):
+    """
+    Load an AIRR repertoire metadata file
+
+    Arguments:
+      file (str): path to the input file.
+      validate (bool): whether to validate data as it is read, raising a ValidationError
+                       exception in the event of an error.
+      debug (bool): debug flag. If True print debugging information to standard error.
+
+    Returns:
+      dictionary of repertoire objects.
+    """
+
+    # Because the repertoires are read in completely, we do not bother
+    # with a reader class.
+    md = None
+
+    # determine file type from extension and use appropriate loader
+    ext = filename.split('.')[-1]
+    if ext == 'yaml' or ext == 'yml':
+        with open(filename, 'r', encoding='utf-8') as handle:
+            md = yaml.load(handle, Loader=yamlordereddictloader.Loader)
```

Comment:
Could be replaced with:

``` 
md = yaml.load(handle, Loader=yaml.FullLoader)
```

If above is done instead of importing `yamlordereddictloader`.